cabot/cabotapp/models.py

```diff
@@ -98,6 +99,12 @@ class Meta:
         blank=True,
         help_text='Users who should receive alerts.',
     )
+    schedules = models.ManyToManyField(
+        'Schedule',
+        blank=True,
+        null=True,
+        help_text='Oncall schedule to be alerted.'
+    )
```

Comment:
I realized that schedules needs to be under CheckGroupMixin since alert() is under CheckGroupMixin. Shouldn't change anything for our use case since we don't use Instances